pkg/workload/core.go

```diff
@@ -155,7 +158,8 @@ func (c *CoreWorkload) Plan(pt JobType, targets []string) error {
 			}
 
 			// If the table is interleaved, find it's entire lineage and add it to the target list
-			if st.IsInterleaved() {
+			if st.IsInterleaved() && !st.IsApex() {
```

Comment:
This is the line I needed to change to fix this issue. `IsInterleaved()` is also true for apex table (root table), so we needed to check if it's not a root table.